File: class25/java/AP2509-SimuladorJuros.java
public class SimuladorJuros {
    public static void main(String[] x) {
        double capital = 1000, taxa = 0.01; int meses = 12;
        System.out.printf("Montante: %.2f%n", capital * Math.pow(1 + taxa, meses));
    }
}

File: class25/cpp/AP2509-SimuladorJuros.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
int main(){double capital=1000,taxa=0.01;int meses=12;std::cout<<std::fixed<<std::setprecision(2)<<"Montante: "<<capital*pow(1+taxa,meses)<<"\n";return 0;}
